Add DataTransferItem getAsString test

xiuqijix opened the pull request:
<!-- Reviewable:start -->

<!-- Reviewable:end -->

xiuqijix committed:
Add DataTransferItem getAsString test

w3c-bots commented:
# Build PASSED

Started: 2017-12-15 02:45:52
Finished: 2017-12-15 02:50:51


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21860)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/316735417)

kaixinjxq commented:
This PR is blocked on the required "Travis CI - Pull Request" check after #14499. In order to trigger it, I will close and reopen this PR.

domenic commented on html/editing/dnd/the-datatransferitem-interface/getAsString-manual.html:
```diff
@@ -0,0 +1,42 @@
+<meta charset="utf-8">
+<title>DataTransferItem Test: getAsString()</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<meta name="flags" content="interact">
```
Not sure what this does.

domenic commented on html/editing/dnd/the-datatransferitem-interface/getAsString-manual.html:
```diff
@@ -0,0 +1,42 @@
+<script src="/resources/testharnessreport.js"></script>
+
+<p><input id="button" type="text" value="dragcharacters" style="border:2px blue solid; width:200px; height: 100px;"/></p>
+<p><input id="div" type="text" style="border:2px green solid; width:200px; height: 100px;"/></p>
```
These IDs are fairly confusing since they are not buttons or divs.

domenic commented on html/editing/dnd/the-datatransferitem-interface/getAsString-manual.html:
```diff
@@ -0,0 +1,42 @@
+let div;
+
+setup(function() {
+  div = document.getElementById("div");
```
This doesn't need to be inside the setup function, right? Might as well put it outside, and just call setup with a single argument.

domenic reviewed (approved):
LGTM with nits

xiuqijix committed:
Address @domenic's comments

- remove meta element which name is flags
- rewrite setup function
- use more appropriate id

kaixinjxq commented:
@domenic, many thanks for your review! I just submit a new commit to address your comments, PTAL.

annevk commented:
Could you rebase this on master due to the Taskcluster configuration change? Sorry :/

kaixinjxq commented:
@domenic, CI checks are all pass, it's ready for merge. :)
